lib/complex.ex

```diff
@@ -1132,17 +1056,17 @@ defmodule Complex do
   Returns a new complex that is the inverse hyperbolic cotangent (i.e., arccoth)
   of the provided parameter.
 
-  #### See also
+  ### See also
   [coth/1](#coth/1)
 
-  #### Examples
+  ### Examples
       iex> Complex.acoth( Complex.from_polar(2,:math.pi) )
       %Complex{im: -8.164311994315688e-17, re: -0.5493061443340548}
 
       iex> Complex.coth( Complex.acoth(Complex.new(2,3)) )
       %Complex{im: 2.999999999999998, re: 2.000000000000001}
   """
-  @spec acoth(complex) :: complex
+  @spec acoth(complex) :: t
   @spec acoth(number) :: number
```

Comment:
I think we need to remove these specs now, no? There are a couple more elsewhere.

```suggestion
  @spec acoth(number) :: number
```

Reply:
You're right. I'll make another pass over those